content/sensu-go/6.4/release-notes.md

```diff
@@ -87,6 +88,19 @@ Read the [upgrade guide][1] for information about upgrading to the latest versio
 
 ---
 
+## 6.4.2 release notes
+
+**August 31, 2021** &mdash; The latest release of Sensu Go, version 6.4.2, is now available for download.
+
+This patch adds a backend configuration flag that allows you to use parallel event log encoding and two summary metrics to the metrics API endpoint.
+
+See the [upgrade guide][1] to upgrade Sensu to version 6.4.2.
+
+**IMPROVEMENTS:**
+
+- ([Commercial feature][215]) Added the [`--event-log-parallel-encoders`][226] backend configuration flag, which allows you to indicate whether Sensu should use parallel JSON encoders for event logging instead of the default (a single JSON encoding worker).
```

Comment:
```suggestion
- ([Commercial feature][215]) Added the [`event-log-parallel-encoders`][226] backend configuration attribute, which allows you to indicate whether Sensu should use parallel JSON encoders for event logging instead of the default (a single JSON encoding worker).
```